x/lsnative/distribution/keeper/delegation.go

```diff
@@ -4,8 +4,9 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/distribution/types"
-	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	sdkdistr "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	sdkstaking "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/lsnative/distribution/types"
```

Comment:
this uses both cosmos-sdk/distribution types and lsnative/distribution types?

Reply:
Yes, the pattern in many files is that most usages of `x/staking`, `x/slashing`, `x/genutil` and `x/distribution` are replaced fully with forked ones, except few cases when we need types from SDK, for example, interfaces.


<img width="872" alt="Screenshot 2023-04-04 at 11 54 36" src="https://user-images.githubusercontent.com/477998/229756291-21ce45aa-0aaf-46c6-9039-ce1b93b98e18.png">

**Context:**
Because these interfaces are often part of iterator callbacks (see an example there https://github.com/persistenceOne/persistenceCore/blob/42c06d7ad27768282696fad66b08991ea2e9dc9c/app/app.go#L341-L344) and if interface will have different type, the whole callback signature needs to be changed and that's way outside the scope.